tests/conftest.py

```diff
@@ -185,10 +206,27 @@ def tmp_metadata_target(tmpdir_factory):
 
 @pytest.fixture
 def netCDFtoZarr_sequential_recipe(
-    daily_xarray_dataset, netcdf_local_paths, tmp_target, tmp_cache, tmp_metadata_target
+    daily_xarray_dataset, netcdf_paths, tmp_target, tmp_cache, tmp_metadata_target
```

Comment:
This is a big change. Now all tests that accepted the old  `netcdf_local_paths` have to also work with multi-variable paths.